lib/ueberauth/strategy/twitter.ex

```diff
@@ -0,0 +1,127 @@
+defmodule Ueberauth.Strategy.Twitter do
+  @moduledoc """
+  Twitter Strategy for Überauth.
+  """
+
+  use Ueberauth.Strategy, uid_field: :id_str
+
+  alias Ueberauth.Auth.Info
+  alias Ueberauth.Auth.Credentials
+  alias Ueberauth.Auth.Extra
+  alias Ueberauth.Strategy.Twitter
+
+  @doc """
+  Handles initial request for Twitter authentication.
+  """
+  def handle_request!(conn) do
+    token = Twitter.OAuth.request_token!([], [redirect_uri: callback_url(conn)])
+
+    conn
+    |> put_session(:twitter_token, token)
```

Comment:
This token is needed after the callback is triggered.  I don't believe private assigns are persisted across requests?